src/test/java/core/basesyntax/DiamondTest.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+
+public class DiamondTest {
+    private static final String ONE_STAR = "* \n";
+
+    private static final String EVEN_NUMBER_OF_STAR =
+                      "   * \n"
+                    + "  * * \n"
+                    + " * * * \n"
+                    + "* * * * \n"
+                    + " * * * \n"
+                    + "  * * \n"
+                    + "   * \n";
+
+    private static final String ODD_NUMBER_OF_STAR =
+                      "    * \n"
+                    + "   * * \n"
+                    + "  * * * \n"
+                    + " * * * * \n"
+                    + "* * * * * \n"
+                    + " * * * * \n"
+                    + "  * * * \n"
+                    + "   * * \n"
+                    + "    * \n";
+
+    @Test
+    public void oneStarScenario() {
+        try {
+            StringStream actualPrintStream = new StringStream(new FileOutputStream("file"));
+            Diamond.drawDiamond(1, actualPrintStream);
+
+            Assert.assertEquals("Test failed with parameter: maxSize = " + 1,
+                    ONE_STAR,
+                    actualPrintStream.getPrintString());
+        } catch (FileNotFoundException e) {
```

Comment:
If test fails with exception in report it'll be shown as passed. We should never ignore exception.